corehq/apps/domain/views.py

```diff
@@ -2209,6 +2232,26 @@ class DomainForwardingRepeatRecords(GenericTabularReport):
         'corehq.apps.reports.filters.select.RepeatRecordStateFilter',
     ]
 
+    def _make_cancel_payload_button(self, record_id):
```

Comment:
There is probably a way, but I think this makes it a bit easier to read (especially since there are only two methods)?  Not sure if that's a valid consideration.